fastify.js

```diff
@@ -9,6 +9,7 @@ const lightMyRequest = require('light-my-request')
 const abstractLogging = require('abstract-logging')
 const proxyAddr = require('proxy-addr')
 
+const symbols = require('./lib/symbols.js')
```

Comment:
Can you use destructuring and avoid passing via `symbols` every time?

Reply:
Do you mean like 

```JavaScript
const {childrenKey, hooksKey} = require('./lib/symbols.js');
```

?

Reply:
Yes exactly.

Reply:
I don't want to do that right now because I think it might be a bit too much work. Sorry. Maybe someone else can jump in on that. I'm ok with that.